splunk_add_on_ucc_framework/ucc_ui_lib/src/main/webapp/components/table/InputTable.jsx

```diff
@@ -0,0 +1,221 @@
+                            appearance="primary"
+                            icon={<Trash screenReaderText={null} size={1} />}
+                            onClick={() => handleDeleteActionClick(row)}
+                        />
+                    </Tooltip>
+                </ButtonGroup>
+            </Table.Cell>
+        )
+    }
+
+    const getTableRow = (row) => {
+        return (
+            <Table.Row 
+                key={row.id} 
+                expansionRow={getExpansionRow(columns.length, row)}
+            >
+                {columns && columns.length && columns.map(
+                    (header) => {
+                        if(header.visible) {
+                            if(header.key === "status") {
+                                return (
+                                    <Table.Cell key={header.key}>
+                                        <Switch
+                                            key={row.name}
+                                            value={row.status}
+                                            onClick={() => handleToggleActionClick(row)}
+                                            selected={!row.status}
+                                            appearance="toggle"
+                                        >
+                                            { row.status ? "Disabled": "Enabled"}
+                                        </Switch>
+                                    </Table.Cell>
+                                )
+                            } else if (header.key == "actions") {
+                                return rowActionsPrimaryButton(row);
+                            } else {
+                                return (
+                                    <Table.Cell key={header.key}>
+                                        {row[header.key]}
+                                    </Table.Cell>
+                                )
+                            }
+                        }
+                    }
+                )}
+            </Table.Row>
+        );
+    }
+
+    const getTableBody = () => {
+        const rows = modifyRowData();
+        return (
+            <Table.Body key={rows}>
+                {rows && rows.length && rows
+                    .sort((rowA, rowB) => {
+                        if (sortDir === 'asc') {
+                            return rowA[sortKey] > rowB[sortKey] ? 1 : -1;
+                        }
+                        if (sortDir === 'desc') {
+                            return rowB[sortKey] > rowA[sortKey] ? 1 : -1;
+                        }
+                        return 0;
+                    })
+                    .map((row) => (
+                        getTableRow(row)
+                    ))
+                }
+            </Table.Body>
+        );
+    }
+
+    return (
+        <>
+            <TitleComponent>Inputs</TitleComponent>
+            <SubTitleComponent>Manage your data inputs</SubTitleComponent>
```

Comment:
Can you please wrap all title and label text with `_()` for i18n & l10n?

Reply:
Done